ctcc_id.sql

```diff
@@ -0,0 +1,11 @@
+-- Adds the CTCC ID validation type
+-- The CTCC ID will always be in the format ‘###-###-###’.
+
+-- Created for University of Florida's PDGENE project
+
+-- Created by:
+-- Laurence Woodley, UF CTSIT
+-- Kyle Chesney, UF CTSIT
+
+REPLACE into redcap_validation_types (validation_name, validation_label, regex_js, regex_php, data_type)
+VALUES ('number_2hyp', 'CHET CTCC ID (###-###-###)', '/^\d{3}-\d{3}-\d{3}$/', '/^\d{3}-\d{3}-\d{3}$/', 'text')
```

Comment:
```suggestion
VALUES ('chet_ctcc_id', 'CHET CTCC ID (###-###-###)', '/^\d{3}-\d{3}-\d{3}$/', '/^\d{3}-\d{3}-\d{3}$/', 'text')
```

Reply:
Are we making the name change? The conversation is resolved but the changes weren't committed.

Reply:
I tried to commit.  I thought I had, but I don't have permission to.  @ljwoodley, could you commit the suggestion?